src/templates/self-serve-form.php

```diff
@@ -49,6 +51,23 @@ class="freq-label listing-single"
 				</label>
 				<div class="input-container listing-details">
 					<p><?php echo wp_kses_post( $single_description ); ?></p>
+					<?php if ( 0 < $single_expiration_period ) : ?>
+						<p class="newspack-listings__help">
+							<?php
+							echo esc_html(
+								sprintf(
+									// Translators: user-facing explanation of expiration behavior.
+									__(
+										'Single-purchase listings expire %d days after the date of publication.',
+										'newspack-listings'
+									),
+									$single_expiration_period
+								)
+							);
+							?>
+						</p>
+					<?php endif; ?>
+					<p class="newspack-listings__help"></p>
```

Comment:
Assuming this empty paragraph is unintended:

```suggestion
```

Reply:
Whoops, you're right!